pom.xml

```diff
@@ -2259,6 +2259,7 @@
               </targetClasses>
               <targetTests>
                 <param>com.puppycrawl.tools.checkstyle.utils.*</param>
+                <param>com.puppycrawl.tools.checkstyle.checks.coding.*</param>
```

Comment:
I moved method from coding check, to utils class. Utils class is in different pitest profile, so it led to downgrade of pitest coverage. I had to add this line and one extra test, to bring it back to 100%